flixel/input/gamepad/ButtonID.hx

```diff
@@ -0,0 +1,57 @@
+package flixel.input.gamepad;
+
+import flixel.system.macros.FlxMacroUtil;
+
+/**
+ * Maps enum values and strings to integer keycodes.
+ */
+@:enum
+abstract ButtonID(Int) from Int to Int
```

Comment:
Sure, I wouldn't mind something shorter. For reference this corresponds to FlxKey in its implementation and use. Got any suggestions?